src/magic_folder/util/encoding.py

```diff
@@ -0,0 +1,4 @@
+import unicodedata
```

Comment:
My fault but this file should probably have:
- copyright header
- docstring (module and for function0
- `from __future__ import (unicode_literals, print_function, absolute_import, division)`